src/components/Tags/Tags.tsx

```diff
@@ -1,26 +0,0 @@
-import clsx from 'clsx';
-import React, { ReactNode } from 'react';
-import styles from './Tags.module.css';
-
-export interface Props {
-  /**
-   * Child node(s) that can be nested inside component
-   */
-  children?: ReactNode;
-  /**
-   * CSS class names that can be appended to the component.
-   */
-  className?: string;
-}
-
-/**
- * Primary UI component for user interaction
- */
-export const Tags = ({ children, className, ...other }: Props) => {
```

Comment:
I think this is handling a separate use case (multiple Tags rather than a single Tag), so would keep it around for now

Reply:
Got it, will bring back `Tags`, might be a little confusing with naming tho

Reply:
ya tru, the naming is unfortunate 🤔  I think our Tag _does_ replace their `Badge` component though, so maybe we can get rid of that?

Reply:
Always down to remove duplicates, will remove `Badge`